app/code/Magento/Catalog/Controller/Adminhtml/Product/SuggestAttributeSets.php

```diff
@@ -46,4 +46,15 @@ public function execute()
         );
         return $resultJson;
     }
+    
+    /**
+     * Check for is allowed
+     *
+     * @return boolean
+     */
+    protected function _isAllowed()
```

Comment:
No need to override Magento\Backend\App\AbstractAction::_isAllowed() as it uses Late Static Binding. More accurate way is to define ADMIN_RESOURCE constant.